floris/tools/floris_interface.py

```diff
@@ -859,7 +846,8 @@ def get_farm_AEP(
         # Copy the full wind speed array from the floris object and initialize
         # the the farm_power variable as an empty array.
         wind_speeds = np.array(self.floris.flow_field.wind_speeds, copy=True)
-        farm_power = np.zeros((self.floris.flow_field.n_wind_directions, len(wind_speeds)))
+        wind_directions = np.array(self.floris.flow_field.wind_directions, copy=True)
+        farm_power = np.zeros(self.floris.flow_field.n_findex)
 
         # Determine which wind speeds we must evaluate in floris
         conditions_to_evaluate = wind_speeds >= cut_in_wind_speed
```

Comment:
It seems that here, we are leaning towards calling the findex the "condition". Not conclusive, but worth noting.

Reply:
Yeah, I drew a blank and used condition, i think it fits well for when different findex are each a different ws/wd, but would be less descriptive is ws/wd matching row to row and changing control settings, or if everything is identical in like a time series,